lib/pip.js

```diff
@@ -201,10 +201,7 @@ function installRequirements(targetFolder, serverless, options) {
     if (options.dockerSsh) {
       // Mount necessary ssh files to work with private repos
       dockerCmd.push(
-        '-v',
```

Comment:
Will this work if all the -v directives are removed? I think you need at least one.

Reply:
Oh, I'm sorry, I copied wrong from my local installation. Thanks!